src/shogun/transfer/multitask/MultitaskKernelMaskPairNormalizer.h

```diff
@@ -58,22 +59,20 @@ class MultitaskKernelMaskPairNormalizer: public KernelNormalizer
 
 	/** initialization of the normalizer
 	 * @param k kernel */
-	virtual bool init(Kernel* k)
+	bool init(Kernel* k) override
 	{
 		ASSERT(k)
 		int32_t num_lhs = k->get_num_vec_lhs();
 		int32_t num_rhs = k->get_num_vec_rhs();
 		ASSERT(num_lhs>0)
 		ASSERT(num_rhs>0)
 
-
 		//same as first-element normalizer
 		auto old_lhs=k->lhs;
 		auto old_rhs=k->rhs;
 		k->lhs=old_lhs;
 		k->rhs=old_lhs;
 
-
 		if (std::string(k->get_name()) == "WeightedDegree") {
```

Comment:
@karlnapf this does nothing afaik. I couldn't find a kernel with this name.. I wanted to change this to a dynamic cast, but I am not sure what this is meant to be...

Reply:
LOL, probably old code?
Remove?

Reply:
I guess.. I assume what this is meant to be is `if (dynamic_cast<WeightedDegreePositionStringKernel*>(k) || dynamic_cast<WeightedDegreeStringKernel*>(k))`, but yea cannot be sure

Reply:
OK, so I did some digging around and at the time this was written that was the name returned by WeightedDegreeStringKernel::get_name() so I’ll fix this!